backend/api/Database/Models/RobotModel.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Api.Database.Models
+{
+    public enum RobotModel
+    {
+        TaurobInspector,
+        TaurobOperator,
+        ExR2,
+        Turtlebot,
+        AnymalX,
+        AnymalD,
+    }
```

Comment:
Should we add a "default" or generic "robot" case? If we want to test something or use the isar-robot repo which is none of these robots.

Reply:
I decided to use the `Turtlebot` as the default since that is our simulator and the one used for testing rather than have a default name `Robot` or similar.

Reply:
So `Turtlebot` is used in mocks and tests etc.

Reply:
Yeah, agreed